plugins/ui/src/js/src/elements/hooks/useDebouncedOnChange.ts

```diff
@@ -0,0 +1,56 @@
+import { useState, useCallback } from 'react';
+import Log from '@deephaven/log';
+import { useDebouncedCallback, usePrevious } from '@deephaven/react-hooks';
+
+const VALUE_CHANGE_DEBOUNCE = 250;
+
+function useDebouncedOnChange<T = string>(
```

Comment:
Don't define the default type, it can be inferred from the value passed in.
```suggestion
function useDebouncedOnChange<T>(
```